plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/summary/block.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "woocommerce/product-summary",
+	"version": "1.0.0",
+	"icon": "page",
+	"title": "Product Summary",
+	"description": "Display a short description about a product.",
+	"attributes": {
+		"productId": {
+			"type": "number",
+			"default": 0
+		},
+		"isDescendentOfQueryLoop": {
+			"type": "boolean",
+			"default": false
+		},
+		"isDescendentOfSingleProductTemplate": {
+			"type": "boolean",
+			"default": false
+		},
+		"isDescendentOfSingleProductBlock": {
+			"type": "boolean",
+			"default": false
+		},
+		"isDescendantOfAllProducts": {
+			"type": "boolean",
+			"default": false
+		},
+		"showDescriptionIfEmpty": {
+			"type": "boolean",
+			"default": false
+		},
+		"showLink": {
+			"type": "boolean",
+			"default": false
+		},
+		"summaryLength": {
+			"type": "number",
+			"default": 0
+		},
+		"linkText": {
+			"type": "string",
+			"default": ""
+		}
+	},
+	"usesContext": [ "query", "queryId", "postId" ],
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"color": {
+			"text": true,
+			"background": true,
+			"link": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"typography": {
+			"fontSize": true
+		},
+		"__experimentalSelector": ".wc-block-components-product-summary"
```

Comment:
Is it necessary?

Reply:
I took it from `supports.ts`. But since I still rely on that file I removed `supports` key completely from `block.json`.